util.py

```diff
@@ -252,3 +252,16 @@ def create_executable(filename, command):
         | stat.S_IXGRP
         | stat.S_IXOTH,
     )
+
+
+def check_file_checksum(filename, reference_checksum):
```

Comment:
if sha256 is hardcoded in the function it should be added to the name (for better readability). i.e. `check_file_sha256sum`